assets/js/watch.js

```diff
@@ -356,6 +356,11 @@ function get_youtube_comments(retries) {
     xhr.send();
 }
 
+function read_more_description(srcButton) {
+    document.getElementById('descriptionWrapper').style.height = "unset";
+    srcButton.style.display = "none";
+}
+
```

Comment:
Please split this and the relevant changes in `watch.ecr` into a separate PR, e.g. "Add 'Read more' to video description".